pkgs/development/python-modules/coapthon3/default.nix

```diff
@@ -11,6 +11,9 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ cachetools ];
 
+  # Disabling tests because they are not included in the package.
+  doCheck = false;
+
```

Comment:
Having disabled tests, consider adding

```nix
  pythonImportsCheck = [ "coapthon" ];
```

which will at least validate that it's possible to import the package.